helm/k8s-node-decorator/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for k8s-node-decorator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# TODO:
+# Change to the image below to be the image in
+# Linode namespace when this project is published.
+
+decorator:
+  # -- The time interval in seconds between 2 polls to the Linode Metadata services by the decorator. 
+  pollInterval: "30"
+
+  image:
+    # -- The decorator container image repository.
+    repository: docker.io/zliangatakamai/k8s-node-decorator
```

Comment:
can we get the docker repo made for this before we commit this PR?

Reply:
So we are going to build automated release workflow in this PR? Sounds good to me!

Reply:
let's gooooo 🚀

Reply:
i literally have no idea how to get that repo made tho, gonna need your hep on that lol

Reply:
https://hub.docker.com/repository/docker/linode/k8s-node-decorator

Reply:
Just added a release workflow: https://github.com/linode/k8s-node-decorator/pull/6/commits/1dc83774a53234c6a71ae5ed984d50707588da95

I tested with my docker hub account and it works fine:
- https://github.com/zliang-akamai/k8s-node-decorator/actions/runs/7468592765
- https://hub.docker.com/repository/docker/zliangatakamai/k8s-node-decorator/tags